packages/api/.mocharc.json

```diff
@@ -8,6 +8,5 @@
   ],
   "watch-files": ["src/**/*.ts", "test/**/*.ts"],
   "recursive": true,
-  "reporter": "spec",
-  "timeout": 200000
```

Comment:
Instead of setting a global timeout for tests I'm now only setting timeouts in the codegen and smoketest suites as they're the only things that can take a while.